Sources/CrossCompilationDestinationsTool/InstallDestination.swift

```diff
@@ -47,7 +47,7 @@ public struct InstallDestination: DestinationCommand {
             bundlePathOrURL: bundlePathOrURL,
             destinationsDirectory: destinationsDirectory,
             self.fileSystem,
-            ZipArchiver(fileSystem: self.fileSystem),
+            UniversalArchiver(self.fileSystem, Cancellator(observabilityScope: observabilityScope)),
```

Comment:
ideally you pass an existing cancelator, eg from swiftTool

Reply:
We don't use `SwiftTool` in destination subcommands as that pulls in a lot of dependencies that aren't needed here, so I hope an ad-hoc executor is fine here?

Reply:
how do you handle cancelling? swift tool has a signal handling routine that then calls the cancellator. ie what happens when users ctrl+c during an install / download sequence?

Reply:
ah, I thought `Cancellator` supported it out of the box. Will have a closer look

Reply:
Addressed in https://github.com/apple/swift-package-manager/pull/6396